app/views/reference/scholarship_types/edit.html.erb

```diff
@@ -0,0 +1 @@
+<%= render 'form' %>
```

Comment:
final newline missing

Reply:
Benzer durumları gelecekte önlemek için VSCode'da `Files: Insert Final Newline`  ayarı yapalım.  (Her editör için buna benzer bir ayar doğrudan veya bir eklenti yoluyla sunulur.  Kullandığınız editörü buna göre ayarlayın.  Şimdilik CI'da lint eklemeyeceğim.)

![image](https://user-images.githubusercontent.com/47842/74417100-3960fc00-4e57-11ea-9c50-487d93981fa3.png)

[Metin dosyalarında neden sonda newline olmalı](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)